nextcord/abc.py

```diff
@@ -1508,6 +1518,8 @@ async def send(
         if file is not None and files is not None:
             raise InvalidArgument("Cannot pass both file and files parameter to send()")
 
+        poll_payload = poll.to_dict() if poll else None
```

Comment:
Could this instead be done like the other `*_payload`s? Like `sticker_payload`/`embed_payload`.